gnomad/resources/grch38/reference_data.py

```diff
@@ -57,6 +57,33 @@ def _import_dbsnp(**kwargs) -> hl.Table:
     return dbsnp
 
 
+def _import_methylation_sites(path) -> hl.Table:
+    """
+    Import methylation data from bed file.
+
+    Methylation scores range from 0-15 and are described in Chen et al
+    (https://www.biorxiv.org/content/10.1101/2022.03.20.485034v2.full).
```

Comment:
would this description be more appropriate where "methylation_sites" is defined since it's more specific to the table resource rather than the function?